azure-pipelines/steps/build_mac.yml

```diff
@@ -6,9 +6,28 @@ steps:
   - bash: |
       brew update
       brew upgrade
+    displayName: 'Update cache and packets'
+  - bash: |
+      xcode-select -s /Applications/Xcode.app
+      brew install --with-clang llvm
+      brew info llvm
+      brew install clang clang++
+      brew info clang
+      brew info clang++
+      echo 'export PATH="/usr/local/opt/llvm/bin:$PATH"' >> ~/.bash_profile
+      export CC=clang
+      export CXX=clang++
+      export LDFLAGS+=" -L/usr/local/opt/llvm/lib -Wl,-rpath,/usr/local/opt/llvm/lib"
+      export CPPFLAGS+=" -I/usr/local/opt/llvm/include -I/usr/local/opt/llvm/include/c++/v1/"
+    #xcode-select --install
+    displayName: 'Install llvm'
```

Comment:
Again the Homebrew Pipeline will be removed shortly. The current master does actually not use it anymore. Thus working on it makes absolutely no sense. Work on [the native pipeline](https://github.com/xournalpp/xournalpp/blob/master/azure-pipelines/steps/build_mac_native.yml) instead.

Reply:
This can also be done in another branch?

Reply:
deferred to pr-freeze/filesystem